apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.GenerateClient/Program.cs

```diff
@@ -33,6 +33,9 @@ class Program
     {
         private const string AppProfileIdFieldName = "_appProfileId";
         private const string AppProfileIdPropertyName = "AppProfileId";
+        private const string CallSettingsTypeName = "CallSettings";
```

Comment:
Use nameof here and for CancellationTokenTypeName?

Reply:
I used it for CancellationToken, but this doesn't have a reference to Gax.Grpc, so I can't use it for CallSettings. Do you think it make sense to add the reference just for this?

Reply:
I don't think the reference would do any harm - on the other hand, it's not like we're going to rename CallSettings. I'm okay either way.

Reply:
Actually, nameof might be better in that it allows a future maintainer of the generator to go to definition and see what's available on that type. And since `nameof(X)` is about the same length as `XTypeName`, I think I'll remove the constants and use nameof in-place (one less go to definition required).